scripts/pre-commit

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# INSTRUCTIONS:
+# This script will run the pre-commit checks in the given order
+# Ktlint
+#
+
+# this will run the ktlint check script
```

Comment:
Sorry I just noticed this -- should we be doing it as a pre-commit check? I think pre-push may be a bit better @BenHenning any thoughts?

Reply:
Ah yes, I agree. Pre-commit is a bit too often I think--let's do pre-push instead.

Reply:
Done